src/Microsoft.NetCore.Analyzers/Core/Security/DoNotUseInsecureDeserializerBinaryFormatterWithoutBinder.cs

```diff
@@ -17,14 +17,14 @@ public class DoNotUseInsecureDeserializerBinaryFormatterWithoutBinder : DoNotUse
         // TODO paulming: Help link URLs.
         internal static readonly DiagnosticDescriptor RealBinderDefinitelyNotSetDescriptor =
             SecurityHelpers.CreateDiagnosticDescriptor(
-                "CA2302",
+                "CA2301",
```

Comment:
Are these all new rules? As in will it break any existing suppressions for these IDs by existing binary FxCop customers?

Reply:
Yep, all new rules.  That's why I wanted to get this change in now, before we ship anything.